### ISSUE TITLE
remote uTorrent not receiving torrent/magnet links 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bogenpirat/remote-torrent-adder#26]
```
What version of Chrome are you using? On what operating system?

Chrome dev 19.0.1.106811 on x86 Windows 7

What client are you connecting to? Are you using SSL?

uTorrent 3.2 Beta 26883 (no SSL) with confirmed working webUI

What steps will reproduce the problem?
1. in RTA Options, select "uTorrent WebUI", enter correct IP, port, login and 
password.
2. verify that remote WebUI works by browsing to it.
3. click on a verified magnet link or torrent link, or "add to remote webUI" 
from dropdown.

What is the expected output? 

magnet link should appear on remote uTorrent client and begin downloading. i 
presume RTA should also display some kind of popup indicating success failure.

What do you see instead?

basically, nothing. there's no success/error popup, and the magnet link doesn't 
register on the remote uTorrent server.

more disturbingly, disabling the remote uTorrent webUI completely also displays 
the same behaviour--there's no "server not available" popup at all, the link 
simply goes nowhere.

disabling RTA results in expected behaviour (torrents/magnets links work, but 
go to local client, not remote).
```

Original issue reported on code.google.com by `speedwaystar` on 17 Mar 2012 at 12:29
